File: decrypt.rb
require 'base64'
require 'mcrypt'

f = open('encrypted_data', 'rb')
encrypted_data = f.read

key = Base64.decode64("6vYJkO5beHNlwOm+aMqUTTzSpRw9jR3faHqu0wVoAG0=")
iv = Base64.decode64('EbuuKuVF4+DzDbWsvQi/ZA==')

crypto = Mcrypt.new('rijndael-128', :cbc, key, iv)
decrypted_data = crypto.decrypt(encrypted_data)

File.open('ruby.jpg', 'w') {|f| f.write(decrypted_data) }

File: decrypt.py

import base64
import mcrypt

f = open('encrypted_data', 'r')
encrypted_data = f.read()

key = base64.b64decode("6vYJkO5beHNlwOm+aMqUTTzSpRw9jR3faHqu0wVoAG0=")
iv = base64.b64decode("EbuuKuVF4+DzDbWsvQi/ZA==")

m = mcrypt.MCRYPT("rijndael-128", "cbc")
m.init(key, iv)
decrypted_data = m.decrypt(encrypted_data)

f2 = open('python.jpg', 'w')
f2.write(decrypted_data)